### PR TITLE
feat: move shows entry point to about tab

### PR DESCRIPTION
**Pull request by MounirDhahri**
The type of this PR is: **Feature**

This PR resolves [CX-743]

### Description
- Move shows entry point to about tab
Screen_Recording_2020-11-27_at_15 07 44

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have added tests for my changes, or my changes don't require testing, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [x] I have added a [CHANGELOG.yml](/CHANGELOG.yml) entry or my changes do not require one.

[CX-434]: https://artsyproduct.atlassian.net/browse/CX-434
[CX-743]: https://artsyproduct.atlassian.net/browse/CX-743

### CODE REVIEW

File: src/lib/Scenes/Artist/Artist.tsx [hunk 1]
```diff
@@ -38,26 +38,28 @@ export const Artist: React.FC<{
...
+      initial: true, // TODO: revert
+      content: artistBelowTheFold ? <ArtistAboutContainer artist={artistBelowTheFold} /> : <LoadingPage />,
     })
   }
 
   if ((artistAboveTheFold.counts?.artworks ?? 0) > 0) {
     tabs.push({
       title: "Artworks",
-      initial: true,
+      initial: false, // TODO: revert
```
Comment (pvinis) [inline src/lib/Scenes/Artist/Artist.tsx:49]: whats up with this one? Should it be reverted in this PR?

Comment (MounirDhahri) [reply src/lib/Scenes/Artist/Artist.tsx]: oh yes sure

File: CHANGELOG.yml [hunk 2]
```diff
@@ -4,6 +4,16 @@ upcoming:
...
+    - Add artist past shows screen - mounir
+    - Add loading spinner to infinite artwork grid on artist page - david
+    - Avoid shrinking the background on non-fullscreen fancy modals - david
+    - Support iOS 14 photo picker in My Collection - brian
+    - Scroll to add message input on Inquire screen - ashley, mounir
+    - Make add message input height relative to its size - ashley, mounir
+    - Avoid shrinking the background on non-fullscreen fancy modals - david
+    - removed x from inquiry sent notification - lily
+    - Minor copy changes for My Collection - pepopowitz
+    - Move shows entry point to about tab - mounir
```
Comment (pvinis) [inline CHANGELOG.yml:16]:
```suggestion
    - Move shows entry point to about tab - mounir
```
I think only this is needed?

**Review by pvinis — CHANGES_REQUESTED** [review]
A couple of cleanup things, then good to go!

File: src/setupJest.ts [hunk 3]
```diff
@@ -107,7 +107,7 @@ mockedModule("./lib/Components/OpaqueImageView/OpaqueImageView.tsx", "AROpaqueIm
 mockedModule("./lib/Components/Artist/ArtistShows/ArtistShows.tsx", "ArtistShows")
 mockedModule("./lib/Components/Artist/ArtistArtworks/ArtistArtworks.tsx", "ArtistArtworks")
 mockedModule("./lib/Components/Artist/ArtistHeader.tsx", "ArtistHeader")
-mockedModule("./lib/Components/Artist/ArtistAbout.tsx", "ArtistAbout")
+// mockedModule("./lib/Components/Artist/ArtistAbout/ArtistAbout.tsx", "ArtistAboutContainer")
```
[the remark this answers is not in the document]
Comment (MounirDhahri) [reply src/setupJest.ts:110]: I implemented the test so i will delete it 🙌

**Pull request merged**